scripts/gee

```diff
@@ -444,7 +444,7 @@ function _check_enkit_cert() {
   if (( COUNT == 0 )); then
     _warn "enkit certificate is expired."
     _info "Please authenticate:"
-    "${ENKIT}" login
```

Comment:
The change from `login` to `auth` here is deliberate?

Reply:
yep -- login and auth are aliases, and I thought it was better to pick one and be consistent (made it easier to grep my code).

Reply:
Makes sense.